Strings/Acknowledgments.jsonc

```diff
@@ -3,33 +3,33 @@
 	"そのほかのライブラリのライセンス条項は端末の設定に移動しました。": {
 		"ja": "そのほかのライブラリのライセンス条項は端末の設定に移動しました。",
 		"en": "Third-party library licenses have been moved to device settings.",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "第三方库的许可协议已移至装置设定",
+		"zh-Hant": "第三方函式庫的授權條款已移至裝置設定",
+		"zh-HK": "第三方函數庫的特許條款已移至裝置設定"
 	},
 
 	"ライブラリ": {
 		"ja": "ライブラリ",
 		"en": "Libraries",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "库",
+		"zh-Hant": "函數庫",
+		"zh-HK": "函式庫"
```

Comment:
By "Libraries" here, I want to refer to third party's open source libraries, is that correct?

Reply:
Yes. That is correct.

Reply:
OK! Thank you!